Src/Plugins/Security/Security.csproj

```diff
@@ -59,8 +55,17 @@
     </Reference>
   </ItemGroup>
 
-  <Target Name="CopyingSpam" AfterTargets="Build">
+  <Target Name="CopyingSecurityToSpam" AfterTargets="Build">
+
+    <ItemGroup>
+      <SecurityFile Include="$(OutputPath)Security.dll" />
+      <SpamFiles Include="$(MSBuildThisFileDirectory)*.json" />
+      <SdkFile Include="$(OutputPath)Sarif.PatternMatcher.Sdk.dll" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(SdkFile)" DestinationFolder="$(SolutionDir).spam\Security\" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false"  ContinueOnError="true" />
```

Comment:
>ContinueOnError="true" [](start = 143, length = 22)

we need this because it will try to find the dll for net472 and netstandard2.0. Since we dont ship net472, lets ignore this.